README.md

```diff
@@ -0,0 +1,10 @@
+Bykr
+====
+
+Installation
```

Comment:
Sure, I'd keep it separate. To look at it another way, this documentation doesn't have much to do with the "Remove geocoding callback from station model" PR.